examples/ng_networking/README.md

```diff
@@ -0,0 +1,59 @@
+# ng_networking example
+
+## Connecting RIOT native and the Linux host
+
+To establish a connection between RIOT and the Linux host, the following dependencies are required:
+
+    sudo apt-get install bridge-utils openvpn
+
+After installing, create a tap interface (to which RIOT will connect) and a bridge (to which Linux will connect):
+
+    sudo brctl addbr br0
+    sudo ip link set br0 up
+    sudo openvpn --mktun --dev tap0
```

Comment:
```bash
sudo ip tuntap add tap0 mode tap user ${USER}
sudo ip link set tap0 up
```

As I said I don't think that the rest is needed, but if you want to create a bridge anyways:

```bash
sudo ip link add name br0 type bridge
sudo ip tuntap add tap0 mode tap user ${USER}
sudo ip link set dev tap0 master br0
sudo ip link set tap0 up
sudo ip link set br0 up
```

It's important to up the interface before the bridge. This way the bridge gets an IPv6 address related to the Hwaddr